src/services/balancer/subgraph/entities/pools/helpers.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { parseUnits } from '@ethersproject/units';
+
+import decoratedPool from './__mocks__/decoratedPool';
```

Comment:
Mocks shouldn't be imported like this. This should be `jest.mock('./decoratedPool')` then whenever that file is imported in any other file during this test the mock will be imported instead.